pipeline-runner/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN echo ".libPaths(c('$RENV_LIB', .libPaths()))" >> $(R RHOME)/etc/Rprofile.sit
 
 # install renv to install required R packages
 RUN R -q -e "install.packages('remotes', repos = c(CRAN = 'https://cloud.r-project.org'))" && \
-    R -q -e "remotes::install_github('rstudio/renv@v1.0.5')" && \
+    R -q -e "remotes::install_github('rstudio/renv@v1.0.7')" && \
```

Comment:
you can point the CRAN repo to the corresponding [posit package manager URL](https://packagemanager.posit.co/client/#/repos/cran/setup?distribution=ubuntu-22.04), which will give you binaries for linux and greatly speed up build time. I think the simplest way here would be to replace it in the lockfiles.

The only caveat here is compatibility with the restore_fast.R script, not sure how they will interact.

Latest r-ver is based on ubuntu jammy, so this is the CRAN url:

https://packagemanager.posit.co/cran/__linux__/jammy/latest

Reply:
bioconductor also provides binaries, but I haven't been able to get those to work nicely with renv :/, which is a bummer, we could be building the images in 10 minutes.

Reply:
You're welcome to look into this as a separate PR. I'm not convinced that it will meaningfully speed up the build (were talking <10 package installs) and, as you mentioned, it may have unintended side effects that need to be debugged.